src/main/java/games/strategy/engine/lobby/client/ui/LobbyFrame.java

```diff
@@ -94,7 +95,7 @@ public ChatMessagePanel getChatMessagePanel() {
   }
 
   private void showServerMessage(final LobbyServerProperties props) {
-    if (props.serverMessage != null && props.serverMessage.length() > 0) {
+    if (!props.serverMessage.isEmpty()) {
```

Comment:
Yes, the server message and error message are never null now. `LobbyServerProperties.java` takes care of that.